src/classes/Recipe.js

```diff
@@ -9,36 +9,40 @@ class Recipe {
   }
 
   matchIngredients(ingredientData) {
```

Comment:
Would love to figure out a way to simplify this once my/our understanding of utilizing iterator methods to combine data sets is a bit more robust.